### PR TITLE
GitHubSync update

### PR DESCRIPTION
**Pull request by particularbot**
This is an automated synchronization PR.

The following source template repositories were used:
* particular/RepoStandards

### CODE REVIEW

File: GitVersion.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
-assembly-versioning-scheme: MajorMinorPatch
-increment: Patch
-next-version: 1.0.2
+assembly-versioning-scheme: Major
```
Comment (andreasohlund) [inline GitVersion.yml]: @bording I think this should be fine right?

**Comment by bording** [comment]
@andreasohlund @ramonsmits This is a repo where I don't know we should have RepoSync enabled. It's still using the old project system and GitVersion 3.x.

File: src/Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<Project>
+
```
Comment (ramonsmits) [inline src/Directory.Build.props:2]: @andreasohlund @bording if we exclude this file we can have reposync enabled I think.

File: .gitignore [hunk 3]
```diff
@@ -36,5 +29,67 @@ _NCrunch_NServiceBus/*
...
+# User-specific files
+*.suo
+*.user
+*.json.lock
+*.nuget.targets
+*.lock.json
+*.userosscache
+*.sln.docstates
 .vs/
-src/PlatformInstaller.ClickOnce/
```
Comment (andreasohlund) [inline .gitignore:40]: We need to check if this needs to still be there

**Pull request closed without merging**